src/app.py

```diff
@@ -1,13 +1,25 @@
 import time
+import eel
+import moisture_reading
 from controller import Controller
 from plant import Plant
 from light import Light
 
+moisture_reading.create_table()
 app = Controller()
+plant = Plant()
 
 print(app.welcome())
 
-while True:
-    moisture_reading = app.soil_moisture(Plant)
-    app.light(moisture_reading, Light)
-    time.sleep(60)
+moisture_reading = app.soil_moisture(Plant)
+app.light(moisture_reading, Light)
+print(moisture_reading)
+
+eel.init('web')
+eel.start('index.html')
+
+# while True:
```

Comment:
Can we remove this loop, rather than comment it out? We can look through older commits if we want it back

Reply:
Sounds good